modules/rest-api/pages/rest-cluster-addnodes.adoc

```diff
@@ -60,12 +60,20 @@ Success gives `200 OK`, and returns an object of the form `{"otpNode":"ns_1@ip-a
 Specifying an unknown service gives `400 Bad Request`, and an object of the form `["Unknown services: [\"unknown-service-name\"]"]`.
 If the node to be added has already been provisioned, and its administrative credentials are not properly specified, `400 Bad Request` is given, and an object is returned of the form `["Prepare join failed. Authentication failed. Verify username and password."]`
 
-If the IP address of the new node is not accurately specified, or otherwise cannot be reached, `400 Bad Request` is given, and an object is returned of the form `["Failed to reach erlang port mapper at node \"ip-address-of-new-node\". Error: ehostunreach"]`.
+If the IP address of the new node cannot be reached, `400 Bad Request` is given, and an object is returned of the form `["Failed to reach erlang port mapper at node \"ip-address-of-new-node\". Error: ehostunreach"]`.
 If the IP address of the host cluster is not accurately specified, or otherwise cannot be reached, the request times out, giving `Empty reply from server`.
 Failure to authenticate with the cluster gives `401 Unauthorized`.
 
 If an attempt is made to specify a node to be added with the scheme `http://` and/or the suffix `8091`, the request fails with `400 Bad Request`, and the error-message `http is prohibited due to security reasons, please use https`.
 
+If the node certificate on the new node does not correctly specify the node-name as a Subject Alternative Name, one of the following occurs:
+
+* A warning is flagged, and the operation succeeds.
+The warning takes the following form: `{"warnings":[{"name":"cert_san_invalid","message":"Address specified in cert SAN part can't be verified.","severity":2,"severityName":"minimal"}]}`.
+
+* An error is flagged, and the operation fails.
+The error takes the following form: `Attention: Prepare join failed. Failed to establish TLS connection to 10.144.231.102:18091. TLS handshake failure (TLS client: In state wait_cert_cr at ssl_handshake.erl:2084 generated CLIENT ALERT: Fatal - Handshake Failure {bad_cert,hostname_check_failed})`.
```

Comment:
This error is not correct. I pointed out the right message on other docs.

Reply:
The error is:

_Attention: Prepare join failed. Unable to validate certificate on host: 127.0.0.1. Please make sure the certificate on this host contains host name '127.0.0.1' in Subject Alternative Name. Refer to Couchbase docs for more info on how to create node certificates._

Reply:
Right, this passage is outdated now. I've added the new message also.